Modules/ExtendedPlayerControl.cs

```diff
@@ -1153,6 +1153,11 @@ public static void ResetKillCooldown(this PlayerControl player)
                     }
                 }
                 break;
+            case CustomRoles.CrewmateTOHE:
+            case CustomRoles.EngineerTOHE:
+            case CustomRoles.ScientistTOHE:
+                Main.AllPlayerKillCooldown[player.PlayerId] = 300f;
```

Comment:
I feel like this would cause issues. Has this been tested?

Reply:
Yes,its buggy.
I gonna move this to check murder